pkg/controller/clusterautoscaler/clusterautoscaler.go

```diff
@@ -34,6 +34,7 @@ func (a AutoscalerArg) TypeRange(t string, min, max int) string {
 // operator when processing ClusterAutoscaler resources.
 const (
 	LogToStderrArg                  AutoscalerArg = "--logtostderr"
+	VerboseArg                      AutoscalerArg = "--v=4"
```

Comment:
I think it is one thing to expose the capability but I'm not sure we should default to such a high level.

Reply:
https://github.com/openshift/cluster-autoscaler-operator/pull/36#discussion_r254350896 huh?

Reply:
I can go back to 3 as I was

Reply:
> [#36 (comment)](https://github.com/openshift/cluster-autoscaler-operator/pull/36#discussion_r254350896) huh?

:) so what I really mean is that if you're trying to debug then 4 yields information that has typically helped me debug - largely because any of the cluster-api cloudprovider logging is at level 4. 

But OOTB I'm asking if we want to default to level 4?

Reply:
The verbose level is set to 3 in many places. Sticking with that here as well.

> But OOTB I'm asking if we want to default to level 4?

I don't think so